test/src/test/java/org/corfudb/runtime/view/QuorumReplicationStreamViewTest.java

```diff
@@ -85,4 +93,10 @@ public void streamWithHoleFill()
         super.streamWithHoleFill();
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testCompactionMark() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651450)